<launch>
     <node name="serial_node"  pkg="serial"  type="serial_node" output="screen">
        <param name="port"                      type="str"    value="/dev/ttyUSB0"/>
        <param name="baudrate"                  type="int"    value="230400"/>
        <param name="max_len"                   type="int"    value="30"/>
        <param name="timeout"                   type="double" value="0.3"/>

        <param name="use_hard_timestamp"        type="bool"   value="false"/>
        <param name="soft_timestamp_dt"         type="double" value="0"/>

        <param name="control/VS_kp"             type="double" value="8"/>
        <param name="control/VS_kd"             type="double" value="0.4"/>
        <param name="control/VS_ff"             type="double" value="0"/>
    </node>

</launch>
